pyproject.toml

```diff
@@ -2,6 +2,9 @@
 name = "nucliadb_workspace"
 requires-python = ">=3.9"
 version = "0.0.0"
+dependencies = [
+    "azure-storage-blob>=12.20.0",
```

Comment:
This dependency should be put in `nucliadb_utils`. Otherwise it won't work for people that download the package from pypi (these are the dependencies for the workspace, which we use for development but it's not published anywhere).